src/callback.jl

```diff
@@ -32,11 +32,16 @@ function jl_Function_call(self_::PyPtr, args_::PyPtr, kw_::PyPtr)
     try
         f = unsafe_pyjlwrap_to_objref(self_)::Function
         if kw_ == C_NULL
-            ret = PyObject(f(convert(PyAny, args)...))
```

Comment:
This package still works on 0.5, so you need a check here e.g. `isdefined(Base, :invokelatest)`.